### ISSUE TITLE
Optimize backend batch create and update

### ISSUE THREAD

**Issue by pepoospina** [uprtcl/js-uprtcl-server#36]
- Create perspectives, ACLs, and initial heads on one query.
- Keep the redundant ecosystem layer updated.
- Keep the finDelegateTo redundant layer updated

[drawing board](https://docs.google.com/drawings/d/1Mdd_dur8Tv0Zu_ZsGF5Bl7f23Hva9XIUIVkJphSP9fI/edit)

**Comment by sotous** [comment]
Hello guys!, finally I got some real progress over here. I managed to have one big upsert for the perspective creation process, all that we do is to call a function that will form that huge string variable for us and send it to the DB instantiation.

So this means that `DGraph` will be able to handle all our creation process, talking about **perspectives**. However, there are still improvements to be made by my side inside this big upsert, and I would like to clarify how this work was achieved:

- The use of the `has` function was implemented, so this means that I still can optimize this upsert by avoiding the use of this function.
- Also, to keep the `finalDelegateTo` updated I assumed that the `finalDelegateTo` will always be `delegate:false` since it is the first-ever created node in the tree, which I call now _the founder_. I use this assumption to query and get the specific result I am looking for to emulate this ACL inside the upsert.

Besides these two previously mentioned points, I should say that performance optimization still is not something greater, I will keep the work going on focusing on the next step, to block out some time for the upsert improvement and of course, keep on track to start working on the `ecosystem`.
